public/app/core/components/AppChrome/MegaMenu/MegaMenu.tsx

```diff
@@ -187,6 +187,8 @@ const getStyles = (theme: GrafanaTheme2) => ({
   }),
   dockMenuButton: css({
     display: 'none',
```

Comment:
@ashharrison90 the `display: 'none'` gets overwritten on inspecting the element and it doesn't look like it's doing anything. Should i remove it?

Reply:
definitely not! 😱 that's the logic that hides the dock button on smaller screen sizes 😉

Reply:
@ashharrison90 ah I get it now, so it's anything above `xl` it will hide. Sorry I totally misread `xl` as very large screens. 

I'm having issues deploying it to an ephemeral instance to test. But once I get that done I'll circle back for a 2nd review.

Reply:
@sleepyfoodie whenever i have an instance failing to build, usually merging main helps. maybe try that 🤞